packages/grafana-data/src/types/datasource.ts

```diff
@@ -322,7 +323,6 @@ export interface ExploreQueryFieldProps<
 > extends QueryEditorProps<DSType, TQuery, TOptions> {
   history: any[];
   onBlur?: () => void;
-  absoluteRange?: AbsoluteTimeRange;
   exploreId?: any;
```

Comment:
This was used for Loki Editor, but we have replaced it with range. I have checked other Explore React Editors and I couldn't find any other use of absoluteRange besides that.